lib/policies/redirectPolicy.ts

```diff
@@ -11,6 +11,27 @@ import {
   RequestPolicyOptionsLike,
 } from "./requestPolicy";
 
+/**
+ * Options for how redirect responses are handled.
+ */
+export interface RedirectOptions {
+  /*
+   * When true, redirect responses are followed.  Defaults to true.
+   */
+  handleRedirects: boolean;
```

Comment:
can we flip this around? I prefer booleans to default to false, so having a "ignoreRedirects" or something would make more sense to me

Reply:
I agree with you, I prefer things that default to false, however I coped this from azure-sdk-for-js, so I left it like this in the interests of compatability. When anyone switches from this library to azure-sdk-for-js, we don't want them to have to invert any redirect options in their calling code. I would suggest we keep this as it is.